go/ql/test/library-tests/semmle/go/frameworks/XNetHtml/test.go

```diff
@@ -2,16 +2,16 @@ package test
 
 import (
 	"database/sql"
-	"golang.org/x/net/html"
 	"net/http"
+	"golang.org/x/net/html"
```

Comment:
```suggestion

	"golang.org/x/net/html"
```
The blank line is need to keep `gofmt` happy. I believe it's between standard library and non-standard library packages.